app/assets/stylesheets/application.scss

```diff
@@ -1,38 +1,13 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any styles
- * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
- * file per style scope.
- *
- *= require_tree .
-
- *= require_self
- */
-
-@font-face {
-    font-family: 'Glyphicons Halflings';
-    src: font-url('../assets/bootstrap/glyphicons-halflings-regular.eot');
-    src: font-url('../assets/bootstrap/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'),
-    font-url('../assets/bootstrap/glyphicons-halflings-regular.woff') format('woff'),
-    font-url('../assets/bootstrap/glyphicons-halflings-regular.ttf') format('truetype'),
-    font-url('../assets/bootstrap/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
-}
-
 $font-family-base: "brandon-grotesque", "helvetica", "sans-serif";
-$font-size-base: 20px;
+$font-size-base: 1.25rem;
 
```

Comment:
The SCSS pipeline broke because bootstrap now uses `rem` and it requires all variables to be in the same unit (rem or px).